### PR TITLE
New branding

### PR DESCRIPTION
**Pull request by Lechu13**

### CODE REVIEW

File: ui/src/components/credentials/CredentialLinkQR.tsx [hunk 1]
```diff
@@ -244,7 +244,7 @@ export function CredentialLinkQR() {
               schemaType={authQRCode.data.linkDetail.schemaType}
               subTitle={
                 <>
-                  Scan the QR code with your Polygon ID wallet to accept it.
+                  Scan the QR code with your Privado iD wallet to accept it.
```
Comment (elias-garcia) [inline ui/src/components/credentials/CredentialLinkQR.tsx:247]: This should be Polygon ID wallet right?

Comment (Lechu13) [reply ui/src/components/credentials/CredentialLinkQR.tsx]: solvedsolved

File: ui/index.html
Comment (amonsosanz) [inline ui/index.html:1]: Apparently, [all the colors have changed recently](https://privadoid.slack.com/archives/C077E9ERSCC/p1719840302178329?thread_ts=1719839453.956219&cid=C077E9ERSCC). Please make sure you are using the right color palette.

**Review by amonsosanz — APPROVED** [review]

File: ui/playwright-report/index.html
Comment (amonsosanz) [inline ui/playwright-report/index.html:1]: I assume this file was committed by mistake right?

**Pull request merged**